logsapi/event.go

```diff
@@ -19,38 +19,44 @@ package logsapi
 
 import (
 	"context"
-	"github.com/elastic/apm-aws-lambda/extension"
-	"github.com/elastic/apm-aws-lambda/apmproxy"
 	"time"
+
+	"github.com/elastic/apm-aws-lambda/apmproxy"
+	"github.com/elastic/apm-aws-lambda/extension"
 )
 
-// EventType represents the type of logs in Lambda
-type EventType string
```

Comment:
[**For reviewers**]: I have renamed this from `EventType` -> `LogType` to avoid confusion with the `EventType` used for lifecycle events. Let me know if it doesn't sound correct.